vim-canvas-demo-app-react/src/components/referral-content/BasicInformation.tsx

```diff
@@ -160,7 +165,17 @@ export const ReferralBasicInformation = () => {
                   authCode: value,
                 },
               })}
-              render={({ field }) => <InputField {...field} />}
+              render={({ field }) => (
+                <TextareaField<ReferralFormInputs>
+                  placeholder="Enter auth code"
+                  control={control}
+                  name={"authCode"}
```

Comment:
Avoid interpolating static strings
```suggestion
                  name="authCode"
```